minimal-app/pom.xml

```diff
@@ -115,7 +116,7 @@
                         </labels>
                     </container>
                     <to>
-                        <image>${docker.registry}/keip/${project.artifactId}</image>
+                        <image>${docker.registry}/${image-name}</image>
```

Comment:
❓ Do we need the version here?

Reply:
Nope, the tag is added to the version. You can see it built and uploaded the correct image tag here.
https://github.com/OctoConsulting/keip/pkgs/container/keip%2Fminimal-app/176007969?tag=0.0.2